pkgs/applications/graphics/textual-paint/default.nix

```diff
@@ -0,0 +1,79 @@
+{ lib
+, python3
+, fetchFromGitHub
+, fetchPypi
+, fetchpatch
+}:
+
+let
+  python = python3.override {
+    packageOverrides = _: super: {
+      pillow = super.pillow.overridePythonAttrs rec {
+        version = "9.5.0";
+        format = "pyproject";
+
+        src = fetchPypi {
+          pname = "Pillow";
+          inherit version;
+          hash = "sha256-v1SEedM2cm16Ds6252fhefveN4M65CeUYCYxoHDWMPE=";
+        };
+
+        patches = [
+          # fix type handling for include and lib directories
+          (fetchpatch {
+            url = "https://github.com/python-pillow/Pillow/commit/0ec0a89ead648793812e11739e2a5d70738c6be5.patch";
+            hash = "sha256-m5R5fLflnbJXbRxFlTjT2X3nKdC05tippMoJUDsJmy0=";
+          })
+        ];
+      };
+
+      textual = super.textual.overridePythonAttrs rec {
+        version = "0.27.0";
+
+        src = fetchFromGitHub {
+          owner = "Textualize";
+          repo = "textual";
+          rev = "v${version}";
+          hash = "sha256-ag+sJFprYW3IpH+BiMR5eSRUFMBeVuOnF6GTTuXGBHw=";
+        };
+      };
+    };
+  };
+in
+
+python.pkgs.buildPythonApplication rec {
+  pname = "textual-paint";
+  version = "0.1.0";
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "1j01";
+    repo = "textual-paint";
+    rev = "v${version}";
+    hash = "sha256-ubBWK4aoa9+wyUED7CmWwjknWsWauR/mkurDgkKDiY8=";
+  };
+
+  nativeBuildInputs = [
+    python.pkgs.setuptools
```

Comment:
Can you also add `python.pkgs.wheel` as well? It will be needed soon after some changes I will be making to Python setup hooks.

Reply:
Will this always be the case after the change? just asking because I might want to change nix-init's behavior

Reply:
Yes when the format is pyproject and setuptools is used as the build backend.

Reply:
https://github.com/nix-community/nix-init/commit/280e59067a5df19ad5e1a26cb845d8cad5cf24d1